pro/src/screens/OfferIndividual/Informations/Informations.tsx

```diff
@@ -0,0 +1,71 @@
+import { useFormik, FormikProvider } from 'formik'
+import React from 'react'
+import { useHistory } from 'react-router-dom'
+
+import useBreadcrumbSteps from 'hooks/useBreadcrumbSteps'
+import Breadcrumb, { BreadcrumbStyle } from 'new_components/Breadcrumb'
+import { OfferIndividualForm } from 'new_components/OfferIndividualForm'
+import { OfferFormLayout } from 'new_components/OfferFormLayout'
+import { CREATION_STEP_PATTERNS } from 'screens/OfferIndividual/constants'
+
+import { ActionBar } from '../ActionBar'
+
+const Informations = (): JSX.Element => {
+  const history = useHistory()
+  const { activeStepId, stepList, getNextStepUrl, getPreviousStepUrl } =
+    useBreadcrumbSteps({
+      stepPatterns: CREATION_STEP_PATTERNS,
+      offerId: null,
+    })
+  const handleNextStep = () => {
+    const testCreatedOfferId = 'AL4Q'
+    history.push(getNextStepUrl(testCreatedOfferId))
```

Comment:
Tu trouverai ca plus simple si ici j'avais : 
```suggestion
    history.push(stepList[STEPS.STOCK].url)
```
?

Et du meme coup je retire les helpers `getStepUrl`, `getPreviousStepUrl` et `getNextStepUrl`